### ISSUE TITLE
Implement a getAttributions method

### ISSUE THREAD

**Issue by geoimpact** [Leaflet/Leaflet#8249]
### Checklist

- [X] I've searched through the [plugins](https://leafletjs.com/plugins.html) to make sure this feature isn't already available, or think it shouldn't require a plugin.
- [X] I've searched through the [current issues](https://github.com/Leaflet/Leaflet/issues) to make sure this feature hasn't been requested already.
- [X] I agree to follow the [Code of Conduct](https://github.com/Leaflet/Leaflet/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.

### Motivation

As a Developer I would like to remove or replace map attributions without needing to do the following:

```javascript
Object.keys(map.attributionControl._attributions).forEach(attribution => {
    map.attributionControl.removeAttribution(attribution);
})
```

### Suggested solution

Provide a public `getAttributions` method for `Control.Attribution`.

I would be happy to submit a PR if this turns out to make sense.

**Comment by Falke-Design** [comment]
@geoimpact yes you can make a `getAttributions` function. I will look over it when you have created a PR.

**Comment by IvanSanchez** [comment]
Attributions belong to `L.Layer`s, not to the attribution control. So

```
map.eachLayer(function(lay) { console.log(lay.getAttribution(); });
```

I don't think attributions should be changed outside of `L.Layer`s. Breaks encapsulation and creates edge cases. I'm really really tempted to close as "won't fix".

**Comment by Falke-Design** [comment]
I think it makes totally sense to get the current text of the attirbution control. Changing will be not working but getting the current text is valid.
